spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -104,7 +104,7 @@
 
       get :timeout
 
-      expect(flash[:timeout]).to eq t('session_timedout')
+      expect(flash[:timeout]).to eq t('session_timedout', app: 'login.gov', minutes: 8)
```

Comment:
So @zachmargolis @jessieay @esgoodman, should we say:
"We signed you out. For your security, APP_NAME automatically ends your session when you’re idle for awhile. Please sign in again."